shared/actions/teams/index.js

```diff
@@ -161,21 +161,27 @@ const _createNewTeamFromConversation = function*(
   }
 
   if (participants) {
-    const createRes = yield call(RpcTypes.teamsTeamCreateRpcPromise, {
-      param: {name, sendChatNotification: true},
-    })
-    for (const username of participants.toArray()) {
-      if (!createRes.creatorAdded || username !== me) {
-        yield call(RpcTypes.teamsTeamAddMemberRpcPromise, {
-          param: {
-            email: '',
-            name,
-            role: username === me ? RpcTypes.TeamsTeamRole.admin : RpcTypes.TeamsTeamRole.writer,
-            sendChatNotification: true,
-            username,
-          },
-        })
+    yield put(Creators.setTeamCreationError(''))
+    try {
+      const createRes = yield call(RpcTypes.teamsTeamCreateRpcPromise, {
+        param: {name, sendChatNotification: true},
+      })
+      for (const username of participants.toArray()) {
+        if (!createRes.creatorAdded || username !== me) {
+          yield call(RpcTypes.teamsTeamAddMemberRpcPromise, {
+            param: {
+              email: '',
+              name,
+              role: username === me ? RpcTypes.TeamsTeamRole.admin : RpcTypes.TeamsTeamRole.writer,
+              sendChatNotification: true,
+              username,
+            },
+          })
+        }
       }
+      yield put(ChatCreators.selectConversation(null, false))
```

Comment:
What does this line do?

Reply:
I couldn't figure out how to dismiss the dialog without this... open to suggestions

Reply:
hm, it seems `yield put(navigateUp())` would work. I'm also getting this behavior where the error stays in the state after the dialog is closed, so it might be a good idea to put a `yield put(Creators.setTeamCreationError(''))` after this line too

Reply:
that's when you run this branch, or master? Note `yield put(Creators.setTeamCreationError(''))` was added at the start of the function in this PR. 
I just used `selectConversation` because that's what new-team-dialog-container.js was doing already.

Reply:
it happened to me on this branch. after I submitted a valid name, there was 1-2s of lag before the dialog went away, and I hit submit again during that time. then the `another team already has this name` error showed up before the dialog closed, and was already there when I opened it up again

Reply:
as for `selectConversation`, that should be fine